src/devices/OneWire/DeviceFamily.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.OneWire
+{
+    /// <summary>
+    /// Different device families supported by the driver.
+    /// </summary>
+    public enum DeviceFamily : int
+    {
+        /// <summary>
+        /// Special device family used when enumerating all devices on a bus.
+        /// </summary>
+        Any = -1,
+        /// <summary>
+        /// Family id of a DS18S20 temperature sensor.
+        /// </summary>
+        Ds18S20 = 0x10,
+        /// <summary>
+        /// Family id of a DS18B20 temperature sensor.
+        /// </summary>
+        Ds18B20 = 0x28,
+        /// <summary>
+        /// Family id of a MAX31820 temperature sensor.
+        /// </summary>
+        Max31820 = 0x28,
+        /// <summary>
+        /// Family id of a DS1825 temperature sensor.
+        /// </summary>
+        Ds1825 = 0x3B,
+        /// <summary>
+        /// Family id of a MAX31826 temperature sensor.
+        /// </summary>
+        Max31826 = 0x3B,
+        /// <summary>
+        /// Family id of a MAX31850 temperature sensor.
+        /// </summary>
+        Max31850 = 0x3B,
+        /// <summary>
+        /// Family id of a DS28EA00 temperature sensor.
+        /// </summary>
+        Ds28EA00 = 0x42,
```

Comment:
There still seem to be some capital letters here and in `Ds18S20`

Reply:
This one I assumed to be a Hexcode, and thought it looked better as EA, do you want it to be Ea because it is a name or ea because it is hex code?

Reply:
In Ds18S20, I went with what I though was the standard, Pascal case, that is first Upper and rest lower, but since only one letter - Upper case S.

Reply:
@krwq I didn't get any feedback on these for next update. Can you just give me the names you want?

Reply:
I'm not sure, I think if it's one symbol then only first letter should be uppercase

Reply:
Ok, I can leave the one letter ones and change EA to Ea in next push.

Reply:
IMO this should be `Ds28ea00` unless `EA00` has any meaning then as you suggested